### PR TITLE
Add ability to add modifiers to the primereact components.

### PR DESCRIPTION
**Pull request by toddburnside**

### CODE REVIEW

File: prime-react/src/main/scala/react/primereact/AccordionMultiple.scala [hunk 1]
```diff
@@ -22,10 +22,14 @@ case class AccordionMultiple(
   onTabOpen:     js.UndefOr[Int => Callback] = js.undefined,
   onTabClose:    js.UndefOr[Int => Callback] = js.undefined,
   onTabChange:   js.UndefOr[Int => Callback] = js.undefined,
-  tabs:          List[AccordionTab] = List.empty
+  tabs:          List[AccordionTab] = List.empty,
+  modifiers:     Seq[TagMod] = Seq.empty
 ) extends ReactFnProps[AccordionBase](AccordionBase.component)
     with AccordionBase {
 
+  def addModifiers(modifiers: Seq[TagMod]) = copy(modifiers = this.modifiers ++ modifiers)
```
Comment (cquiroz) [inline prime-react/src/main/scala/react/primereact/AccordionMultiple.scala:30]: Could these be part of `ReactFnProps`? or another common trait?

**Review by cquiroz — COMMENTED** [review]
LGTM

File: prime-react/src/main/scala/react/primereact/AccordionMultiple.scala (hunk 1 above)
Comment (rpiaggio) [reply prime-react/src/main/scala/react/primereact/AccordionMultiple.scala:30]: I think that the trait would have to know how to `copy`. Or use a mutable accumulator.

BTW, could we have a `def apply(mods: TagMod*)` method?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: prime-react/src/main/scala/react/primereact/AccordionMultiple.scala (hunk 1 above)
Comment (toddburnside) [reply prime-react/src/main/scala/react/primereact/AccordionMultiple.scala:30]: > I think that the trait would have to know how to `copy`. Or use a mutable accumulator.

Right. I don't see a way around this. So, I don't think there is a lot of benefit to having a trait.

**Pull request merged**